certbot/plugins/dns_common_lexicon.py

```diff
@@ -7,6 +7,18 @@
 from certbot import errors
 from certbot.plugins import dns_common
 
+# Lexicon is not declared as a dependency in Certbot itself,
+# but in the Certbot plugins backed by Lexicon.
+# So we catch import error here to allow this module to be
+# always importable, even if it does not make sens to use it
+# if Lexicon is not available, obviously.
+# This import is not done at runtime to allow mypy type to
+# be handled correctly in any situation.
```

Comment:
I don't understand this last sentence. We may not want to do it for other reasons, but bad thing would happen with mypy if we moved this import into the body of `build_lexicon_config`?

Reply:
Yes, in this case `ConfigResolver` would not be available to mypy and type check would fail. But indeed it the natural way to make import anyway, so I remove this phrase.